### ISSUE TITLE
Automate Smart Tags into CMS

### ISSUE THREAD

**Issue by danielabloch** [Parsely/wp-parsely#1072]
Automate Smart Tags Back Into Customer CMS for SEO

## Is your feature request related to a problem?
In a recent meeting with Daily Mail, Carlo recalled a workflow that Neil Powell implemented at S&P. He would look at their content (maybe a subset) and compare keywords editorial tagged to Smart Tags. He would then add the Smart Tags in the CMS to improve SEO but also presumably to make it easier to find content in the CMS itself.

From Neil: 

> We reviewed smart tags from Parsely and added a few of the really topical ones to our custom CMS that had a lot of stories/usage. Then they could be found in our platform's internal search capability. If I remember correctly, it was some of our subject tags around ESG coverage.

I assume any customer could automate this process currently using the API but it could be cool if this was offered as a feature of the WP Plugin.

## Describe the solution you'd like
Ability to surface Smart Tags being captured in the Parse.ly dashboard into the WordPress plugin for Parse.ly. Customers could then add those tags to their content directly or with a click of a button. 

## Describe alternatives you've considered
None.

## Additional context
We should discuss this with Neil Powell directly. 

## Notes
- This would be the dream for content creators. No content creator likes tagging content, humans are bad at tagging consistently, and there's a ton of time savings that can be done when automating tagging. Would you rather create more content or tag a story?
- Interesting. It would be great if the Smart Tags layer could exist in WP without any action needed by editors. Like they just showed up alongside Tags automatically. You could add them as "real" Tags if you wanted. But even if you didn't, they would be there as a way to navigate content in the CMS

**Comment by GaryJones** [comment]
Tagging has further implications beyond internal search, since it then makes an article appear on that terms's archive page as well. A customer may already have an agreed set of terms/tags that should be used, so to start creating more may disrupt that. For instance, maybe an existing term is `seo`, but the added smart tag would be `search-engine-optimization`, or maybe it's a UK site, and they have `search-engine-optimisation`, but Parse.ly adds another tag with the US spelling.

Rather than just go ahead and add tags, they could maybe be a preview of tags that would be added: "Hey, Parse.ly has identified X, Y, and Z as popular possible tags. Would you like to add these tags to your posts?"

**Comment by chriszarate** [comment]
> Rather than just go ahead and add tags

Agreed, we're not considering automatically adding tags without a user decision. Rather we'll make that decision easy via UI elements.

**Comment by GaryJones** [comment]
Coincidentally, a customer highlighted in a call yesterday that tag management, and the need to restrict new ones being added, was a concern for them, when Parse.ly's smart tags feature was mentioned.

**Comment by danielabloch** [comment]
> Coincidentally, a customer highlighted in a call yesterday that tag management, and the need to restrict new ones being added, was a concern for them, when Parse.ly's smart tags feature was mentioned.

@GaryJones which customer was that and would they be available to meet with me to talk about their process for publishing content? are they Parse.ly customers?
